pkg/writer/pointerdos_test.go

```
@@ -0,0 +1,50 @@
+		data, top := buildPointerFanOutData(depth)
+		got := countLeafDecodes(data, top)
+		want := 1 << depth
+		if got != want {
+			t.Errorf("depth %d: leaf decoded %d times, want %d", depth, got, want)
+		}
+	}
+}
+
+func TestPointerFanOutFixturesMatchCommitted(t *testing.T) {
+	cases := map[string][]byte{
+		"MaxMind-DB-test-pointer-decoder-dos.mmdb":      buildPointerFanOutDB(pointerDoSDepth),
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(golines)